trajopt_ifopt/src/constraints/cartesian_position_constraint.cpp

```diff
@@ -193,27 +193,27 @@ CartPosConstraint::CartPosConstraint(CartPosInfo info,
   }
 }
 
-CartPosConstraint::CartPosConstraint(CartPosInfo info,
+CartPosConstraint::CartPosConstraint(const CartPosInfo& info,
                                      std::shared_ptr<const JointPosition> position_var,
                                      const std::string& name)
-  : CartPosConstraint(std::move(info), std::move(position_var), Eigen::VectorXd::Ones(info.indices.rows()), name)
+  : CartPosConstraint(info, std::move(position_var), Eigen::VectorXd::Ones(info.indices.rows()), name)
```

Comment:
Good catch, did clang-tidy find this issue?

Reply:
Yes, clang-tidy is great for that!